js/components/planet-wizard.tsx

```diff
@@ -26,7 +28,18 @@ interface IStepsData {
   info: (geode: boolean) => string; // label of bottom bar button
   navigationDisabled?: boolean; // whether next/back navigation should be disabled globally
   nextDisabled?: (simulationStore: SimulationStore) => boolean; // whether next navigation should be disabled conditionally
+  validation?: IValidation; // when defined, the test test will be ran before user can go to the next step
 }
+
+interface IValidation {
+  // If this function returns false, user will see a warning dialog that lets him return and fix the configuration.
```

Comment:
```suggestion
  // If this function returns false, user will see a warning dialog that lets them return and fix the configuration.
```
An opportunity to use an inclusive pronoun.